crates/accelerate/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "qiskit_accelerate"
+# The following options can be inherited with (e.g.) `version.workspace = true` once we hit Rust
+# 1.64.  Until then, keep in sync with the root `Cargo.toml`.
+version = "0.24.0"
+edition = "2021"
+rust-version = "1.61"
+license-file = "LICENSE.txt"
```

Comment:
Does this work still because there isn't a local `LICENSE.txt` in the crate dir?

Reply:
Oh, I should check - it's a copy paste bug from when I discovered that the workspace inheritance is Rust 1.64+.

Reply:
I cheated and just set `license` as Apache 2 in a698245.  Probably a better choice overall anyway.